### PR TITLE
[RORDEV-616] ROR ECK quickstart improvements

### PR DESCRIPTION
**Pull request by coutoPL**
- ROR installed from API
- Kibana HA mode enabled
- clean up

### CODE REVIEW

File: docker-envs/eck/kind-cluster/ror/kbn.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          env:
+            - name: KBN_VERSION
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.labels['kibana.k8s.elastic.co/version']
+          volumeMounts:
+            - name: kibana-plugins
+              mountPath: /usr/share/kibana/plugins
+            - name: kibana-node-modules-kbn
+              mountPath: /target/usr/share/kibana/node_modules/@kbn
```
Comment (coutoPL) [inline docker-envs/eck/kind-cluster/ror/kbn.yml:52]: @Dzuming please confirm that we change files only in these two locations: `node_modules/@kbn` and `src/cli`

Comment (Dzuming) [reply docker-envs/eck/kind-cluster/ror/kbn.yml:52]: As I see, we also patch:
- [src/legacy](https://github.com/sscarduzio/readonlyrest_kbn/blob/e4f8c579b75a48a2f17a30fbc57db00895ca38cf/kibana/patchers/patches_for_kbn_distribution/v7.9.x/kbn_server.js.patch)
- [src/core](https://github.com/sscarduzio/readonlyrest_kbn/blob/d48328e1e9df035e36d550adf5bfafe17d4961fb/kibana/patchers/patches_for_kbn_distribution/v8.4.x/server.patch)
- [x-pack/plugins](https://github.com/sscarduzio/readonlyrest_kbn/blob/18802f370ca48505fea184d99bd34ea8e42a2756/kibana/patchers/patches_for_kbn_distribution/v8.5.x/event_log_client.patch)

Comment (coutoPL) [reply docker-envs/eck/kind-cluster/ror/kbn.yml:52]: I didnt see these in the patcher logs. I've tested KBN 8.11.

Comment (Dzuming) [reply docker-envs/eck/kind-cluster/ror/kbn.yml:52]: We keep poacher files for specific versions here https://github.com/sscarduzio/readonlyrest_kbn/tree/develop/kibana/patchers/patches_for_kbn_distribution

Comment (Dzuming) [reply docker-envs/eck/kind-cluster/ror/kbn.yml:52]: If you need patch files only for 8.11.3, the list you provided is correct

Comment (coutoPL) [reply docker-envs/eck/kind-cluster/ror/kbn.yml:55]: ok, I've updated the mount paths

**Pull request merged**